iroh/src/protocol.rs

```diff
@@ -300,12 +302,28 @@ impl RouterBuilder {
                                     break;
                                 }
                             }
-                            Some(Ok(Err(inner))) => {
-                                debug!("Task errored: {inner:?}");
+                            Some(Ok(Some(()))) => {
+                                debug!("Task finished");
+                            }
+                            Some(Ok(None)) => {
+                                debug!("Task cancelled");
```

Comment:
I'm not sure if we should log this... I mean, it doesn't cost *much*, but I do think there's potential for needless log pollution.

Reply:
switched to `trace`